client/lib/RoomManager.coffee

```diff
@@ -105,16 +105,14 @@ Meteor.startup ->
 		return room?.dom?
 
 	updateUserStatus = (user, status, utcOffset) ->
-		onlineUsersValue = onlineUsers.curValue
+		allUsersValue = allUsers.curValue
```

Comment:
I dunno.  I just changed their currrent one from online.  You're much more familiar with it.